src/main/java/uk/gov/digital/ho/hocs/info/client/ingest/ListConsumerService.java

```diff
@@ -125,14 +117,14 @@ public ListConsumerService(@Value("${api.uk.parliament}") String apiUkParliament
 
         ResponseEntity<T> response = restTemplate.exchange(apiEndpoint, HttpMethod.GET, entity, returnClass);
 
-        if (response == null || response.getStatusCodeValue() != 200) {
+        if (response.getStatusCodeValue() != 200) {
```

Comment:
Why is the null check not needed?

Reply:
restTemplate.exchange has a annotated return with @notnull which means response can never be null at this stage